README.md

```diff
@@ -14,7 +17,16 @@ In this project you will find:
 
 ## Local development
```

Comment:
I think we should get rid of the local development part, as its in the docs too?